src/internal/fixtures.ts

```diff
@@ -4,4 +4,7 @@ export const SEPARATORS_TEXT =
 export const WEIRD_TEXT =
   ' someWeird-cased$*String1986Foo [Bar] W_FOR_WUMBO...' as const
 
-export type WeirdTextUnion = typeof WEIRD_TEXT | 'dont.distribute unions'
+export type WeirdTextUnion =
+  | typeof WEIRD_TEXT
+  | "where's the leak ma'am"
```

Comment:
Where's the leak ma'am?
![image](https://github.com/gustavoguichard/string-ts/assets/33426811/c66c5dc8-b982-41b4-85af-f185c0b54dca)